src/preamble.js

```diff
@@ -603,14 +599,14 @@ function findWasmBinary() {
 #endif
 
 function getBinarySync(file) {
+#if SINGLE_FILE
+  if (ArrayBuffer.isView(file)) {
+    return file;
+  }
```

Comment:
Isn't it always a view here, so can't we remove the `if`?

Reply:
This code I believe can be used to load other wasm files, not just the main wasm file.   Unless I'm misremembering.

Reply:
Oh, maybe in dynamic linking? sgtm.

Reply:
Actually, no it looks like only `emscripten_lazy_load_code` and `load_secondary_module` use these loading functions for other wasm file..   anyway, maybe that can be a diffident cleanup.

Reply:
Sounds good.